web/themes/custom/novel/templates/fields/field--field-more-link.html.twig

```diff
@@ -0,0 +1,5 @@
+{% for item in items %}
```

Comment:
The file is `field--field-more-link.html.twig`, but the file includes references to `card-grid` - can you target the paragraph specificially?
Like field--field--more--link--paragraph--blabla or something?